interfaces/saml/v0/README.md

```diff
@@ -0,0 +1,75 @@
+# `saml`
+
+## Overview
+
+This relation interface describes the expected behavior between of any charm claiming to be able to interface with a SAML Provider and the SAML Provider itself. This interface has been defined to support charms that require Launchpad group information. Other Identity Providers can be used interchangeably as long as custom attributes are not required.
+
+## Usage
+
+In most cases, this will be accomplished using the [saml library](https://github.com/canonical/saml-integrator-operator/blob/main/lib/charms/saml_integrator/v0/saml.py), although charm developers are free to provide alternative libraries as long as they fulfil the behavioural and schematic requirements described in this document.
+
+## Direction
+
+The `saml` interface implements a provider pattern.
```

Comment:
What is a provider pattern? Don't we mean `provider/requirer`?

Reply:
Fixed. Thx